### PR TITLE
ADM-796:[backend]feat: set prLeadTime to 0 when commit is reverted

### PR DESCRIPTION
**Pull request by PengxiWPix**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by WSSsssss33 — APPROVED** [review]
LGTM

File: backend/src/main/java/heartbeat/service/source/github/GitHubService.java [hunk 1]
```diff
@@ -232,11 +232,17 @@ public LeadTime mapLeadTimeWithInfo(PullRequestInfo pullRequestInfo, DeployInfo
 		long pipelineLeadTime = jobFinishTime - prMergedTime;
 		long prLeadTime;
 		long totalTime;
-		if (firstCommitTimeInPr > 0) {
-			prLeadTime = prMergedTime - firstCommitTimeInPr;
+		if (commitInfo.getCommit().getMessage().startsWith("Revert")
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/service/source/github/GitHubService.java:235]: maybe `commitInfo.getCommit().getMessage().toLowcase()` is a good way

File: backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java [hunk 2]
```diff
@@ -274,9 +275,51 @@ void CommitTimeInPrShouldBeZeroWhenCommitInfoIsNull() {
 		assertEquals(expect, result);
 	}
 
+	@Test
+	void CommitTimeInPrLeadTimeShouldBeZeroWhenCommitInfoIsRevert() {
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java:279]: @PengxiWPix 
update test description to `should ....when...given`, not start a customized message

File: backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java [hunk 3]
```diff
@@ -274,9 +275,51 @@ void CommitTimeInPrShouldBeZeroWhenCommitInfoIsNull() {
...
+			.totalTime(120000)
+			.build();
+
+		LeadTime result = githubService.mapLeadTimeWithInfo(pullRequestInfo, deployInfo, commitInfo);
+
+		assertEquals(expect, result);
+	}
+
+	@Test
+	void CommitTimeInPrLeadTimeShouldBeZeroWhenCommitInfoIsRevertInLowerCase() {
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java:300]: same as the above

**Pull request merged**